src/utils/SnapshotProvider.js

```diff
@@ -62,7 +63,7 @@ const getProposals = async first => {
 };
 
 const getVoteByVoter = async (address, proposalId) => {
-  const { data } = await axios.post(snapshotGraphqlEndpoint, {
+  const { data } = await axios.post(`${snapshotBaseEndpoint}graphql`, {

```

Comment:
I would suggest that we dont add the "/" in the config values but at the code level. so we have the path in the function and the other part of the url in the config

so in the env config: scheme + host and in the code: path + query params

Reply:
Thanks